RELEASE/scripts/autoscend/auto_adventure.ash

```diff
@@ -1,5 +1,34 @@
 script "auto_adventure.ash"
 
+// just a wrapper around https://svn.code.sf.net/p/therazekolmafia/canadv/code/ for now, but gives a single point of change if we need to change how we determine this
+boolean autoCanAdv(location loc){
+
+	static boolean[item] craftable_shirts = $items[barskin cloak, bat-ass leather jacket, clownskin harness, demonskin jacket, gnauga hide vest, hipposkin poncho, lynyrdskin tunic, tuxedo shirt, white snakeskin duster, yak anorak];
+
+	boolean acquire_a_shirt(){
+		if(isjanuaryToteAvailable() && januaryToteAcquire($item[Makeshift Garbage Shirt])){
+			return true;
+		}
+		if(auto_have_skill($skill[Torso Awaregness]) && auto_have_skill($skill[Armorcraftiness])){
+			foreach shirt in craftable_shirts{
+				if(creatable_amount(shirt) > 0 && create(1, shirt)){
+					return true;
+				}
+			}
+		}
+		return false;
+	}
+
+	if(!can_adv(loc)){
+		if(loc == $location[The Thinknerd Warehouse] && !acquire_a_shirt()){
```

Comment:
should probably try using a shirt we might already own e.g. one of the four from the swarm of scarab beatles, shoe ad t-shirt from the Neverend, red shirt & red coat from the red box from Red Zeppelin enemies, surgical apron & midriff scrubs from the Hidden Hospital, unrequired jacket from Copperhead etc. as most of those craftings will cost an adventure IIRC.

Reply:
We'll i think the problem is if we had gotten one of those we would already have the quest/location right? Or does it trigger on equipping a shirt? if so I can loop through all shirts and equip one if we got it. I also did consider using free adventures in NEP to try to get one, but those are pretty valuable, not sure we want to use them for this.

Reply:
It triggers on equipping the shirt IIRC. Just having it in inventory isn't enough, you need to attempt to equip one regardless of whether you have Torso Awaregness.

Reply:
Yeah I think thats right. Ill modify it to look for and equip existing shirts before trying to acquire one.